index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+
+        <header class="header-container">
+            <h3 class="white">My Portfolio</h3>
```

Comment:
- [Optional] Consider wrapping your logo in an `a` tag with href attribute that points to index html. This is best practice for HTML semantics and also for good user experience.